src/Auth0Client.ts

```diff
@@ -19,7 +19,10 @@ import TransactionManager from './transaction-manager';
 import { verify as verifyIdToken } from './jwt';
 import { AuthenticationError, GenericError } from './errors';
 import * as ClientStorage from './storage';
-import { DEFAULT_POPUP_CONFIG_OPTIONS } from './constants';
+import {
+  DEFAULT_POPUP_CONFIG_OPTIONS,
```

Comment:
unless `DEFAULT_POPUP_CONFIG_OPTIONS` is being used somewhere else, let's remove this

Reply:
This one is being used as well. What is making you think that they're not? Is there an issue with the diff?

Reply:
The diff just shows that you're removing one usage. But if you open the entire file, you see it's being used somewhere else. So dismiss my comment.